### PR TITLE
Vision v1p1beta1 API

### PR DESCRIPTION
**Pull request by jskeet**
There are a couple of changes to other APIs in here due to regeneration, but in separate commits. They're harmless.

Still to do (but not blocking a first beta package release):

- Check whether any of the new features need handwritten support
- Add details to the package-naming document about what "V1P1Beta1" means
- Possibly add a "this is beta" to the V1P1Beta1 index.md, linking to V1?

### CODE REVIEW

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.IntegrationTests/ImageAnnotatorClientTest.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.IntegrationTests/ImageAnnotatorClientTest.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/BatchAnnotateImagesResponseSnippets.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/BatchAnnotateImagesResponseSnippets.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/ImageAnnotatorClientSnippets.cs [hunk 3]
```diff
@@ -0,0 +1,345 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/ImageAnnotatorClientSnippets.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/ImageSnippets.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/ImageSnippets.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/Rectangle.cs [hunk 5]
```diff
@@ -0,0 +1,77 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Snippets/Rectangle.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/AnnotateImageResponseTest.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/AnnotateImageResponseTest.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/BatchAnnotateImagesResponseTest.cs [hunk 7]
```diff
@@ -0,0 +1,76 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/BatchAnnotateImagesResponseTest.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/FakeImageAnnotator.cs [hunk 8]
```diff
@@ -0,0 +1,88 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/FakeImageAnnotator.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/ImageAnnotatorClientTest.cs [hunk 9]
```diff
@@ -0,0 +1,226 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/ImageAnnotatorClientTest.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/ImageTest.cs [hunk 10]
```diff
@@ -0,0 +1,142 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1.Tests/ImageTest.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/AnnotateImageResponsePartial.cs [hunk 11]
```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/AnnotateImageResponsePartial.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/BatchAnnotateImagesResponsePartial.cs [hunk 12]
```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/BatchAnnotateImagesResponsePartial.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImageAnnotationException.cs [hunk 13]
```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImageAnnotationException.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImageAnnotatorClientPartial.cs [hunk 14]
```diff
@@ -0,0 +1,490 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImageAnnotatorClientPartial.cs]: 2017?

File: apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImagePartial.cs [hunk 15]
```diff
@@ -0,0 +1,210 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Vision.V1P1Beta1/Google.Cloud.Vision.V1P1Beta1/ImagePartial.cs]: 2017?

**Comment by jskeet** [comment]
Thanks, will fix all :)

**Pull request merged**